src/patternfly/_chart-globals.scss

```diff
@@ -256,6 +256,7 @@ $pf-chart-stack--data--stroke--Width: $pf-chart-global--stroke--Width--xs;
 
 // Threshold
 $pf-chart-threshold--stroke-dash-array: "6,6";
```

Comment:
Were you going to change this to `4,2` in this PR?

Reply:
Updated, thank you